src/adapters/acala.ts

```diff
@@ -210,22 +210,16 @@ class BaseAcalaAdapter extends BaseCrossChainAdapter {
     }
 
     const accountId = this.api?.createType("AccountId32", address).toHex();
-
-    // to other parachains
-    const dst: any = {
-      parents: 1,
-      interior: {
-        X2: [
-          { Parachain: toChain.paraChainId },
-          { AccountId32: { id: accountId, network: "Any" } },
-        ],
-      },
-    };
+    const dst = xTokensHelper.buildV1orV3Destination(
+      this.api,
+      accountId,
+      toChain
+    );
 
     return this.api.tx.xTokens.transfer(
       tokenFormSDK?.toChainData() as any,
       amount.toChainData(),
-      { V1: dst },
+      dst as any,
```

Comment:
This is really a nit, so feel free to ignore, but we could simplify event more, right?  Instead of..
```
const destWeight = ...
const dst = ...
return this.api.tx.xTokens.transfer
```
Could we not have this? 
```
return xTokensHelper.transfer(this.getDestWeight(token, to), accountId, toChain)
```

Reply:
That's a good shout, thank you.
I agree that would simplify it even further. On it.

Reply:
Done in [commit 2754c94](https://github.com/interlay/bridge/pull/85/commits/2754c944c7d5e11cc12f3309a4c2246cd050bc40)